vmicore/src/lib/plugins/PluginSystem.h

```diff
@@ -17,6 +17,8 @@
 
 namespace VmiCore
 {
+    constexpr std::string_view PLUGIN_INIT_FUNCTION = "init_plugin";
```

Comment:
Just a suggestion. No need to add it if you don't want to, but maybe name it something like "vmicore_plugin_init" since we are already at it?